tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -127,6 +156,16 @@ def test_saas_access_request_task(
         )
         > 0
     )
+    assert (
```

Comment:
Thanks for adding this new use case, I don't think these checks are too important, you'll see we stopped doing this for some of the newer connections.

Reply:
removed!